src/main/java/com/redhat/documentation/asciidoc/extraction/Assembly.java

```diff
@@ -46,10 +46,7 @@ public Assembly(Document doc, List<String> lines) {
         this.createAssembly = Boolean.parseBoolean(doc.getAttribute("assembly", "true").toString());
 
         // Adding the id of the module
-        this.source.append("[id=\"").append(this.idWithoutContext).append("_{context}\"]\n")
-                // Adding the section title
-                .append("= ").append(doc.getTitle()).append("\n")
-                .append(":context: ").append(this.idWithoutContext).append("\n\n");
+        this.source.append("[id=\"").append(this.idWithoutContext).append("_{context}\"]\n");
```

Comment:
Why??

Reply:
the title and context lines were getting repeated

Reply:
Got it